tests/epyccel/test_epyccel_dicts.py

```diff
@@ -181,3 +181,15 @@ def get_default_str_keys():
     python_result = get_default_str_keys()
     assert isinstance(python_result, type(pyccel_result))
     assert python_result == pyccel_result
+
+def test_dict_clear(language):
+    def dict_clear():
+        a = {1:1.0, 2:2.0}
+        a.clear()
+        return a
+    epyc_dict_clear = epyccel(dict_clear, language = language)
+    pyccel_result = epyc_dict_clear()
+    python_result = dict_clear()
+    assert len(python_result) == len(pyccel_result)
+    assert len(python_result) == 0
+    assert len(pyccel_result) == 0
```

Comment:
You don't need to use `len` outside the function that is translated. You are already returning the dictionary (this is ok as you are only translating to Python) so you can simply do:
```suggestion
    assert python_result == pyccel_result
```